otelcli/config.go

```diff
@@ -79,9 +80,10 @@ type Config struct {
 	TraceparentPrintExport bool   `json:"traceparent_print_export" env:"OTEL_CLI_EXPORT_TRACEPARENT"`
 	TraceparentRequired    bool   `json:"traceparent_required" env:"OTEL_CLI_TRACEPARENT_REQUIRED"`
 
-	BackgroundParentPollMs int    `json:"background_parent_poll_ms" env:""`
-	BackgroundSockdir      string `json:"background_socket_directory" env:""`
-	BackgroundWait         bool   `json:"background_wait" env:""`
+	BackgroundParentPollMs       int    `json:"background_parent_poll_ms" env:""`
+	BackgroundSockdir            string `json:"background_socket_directory" env:""`
+	BackgroundWait               bool   `json:"background_wait" env:""`
+	BackgroundSkipParentPidCheck bool   `json:"background_skip_pid_check"`
```

Comment:
Ideally the json will match the variable name here, to be consistent. Usually I prefer to shorten the variable name and let the json key be as long as it takes to be clear. Please switch it to `background_skip_parent_pid_check` and I'll be happy to merge.

Reply:
Sure, so I switched it to `background_skip_parent_pid_check`